### PR TITLE
Add WorkerNavigator.mediaCapabilities

### PR DESCRIPTION
**Pull request by teoli2003**
Part of openwebdocs/project#152

- Adds `WorkerNavigator.mediaCapabilities`
- List it in `WorkerNavigator`
- Remove bogus syntax box in `Navigator.mediaCapabilities`

### CODE REVIEW

File: files/en-us/web/api/navigator/mediacapabilities/index.md [hunk 1]
```diff
@@ -13,17 +13,11 @@ browser-compat: api.Navigator.mediaCapabilities
 
 {{APIRef("HTML DOM")}}
 
-The **`Navigator.mediaCapabilities`** read-only property
+The read-only **`mediaCapabilities`** property of the {{domxref("Navigator")}} interface
```
Comment (hamishwillee) [inline files/en-us/web/api/navigator/mediacapabilities/index.md:16]: The original format is better, and is used in a standard way across MDN.
There is a link to the parent in the breadcrumb too.

File: files/en-us/web/api/workernavigator/mediacapabilities/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+title: WorkerNavigator.mediaCapabilities
+slug: Web/API/WorkerNavigator/mediaCapabilities
+page-type: web-api-instance-property
+browser-compat: api.WorkerNavigator.mediaCapabilities
+---
+
+{{APIRef("HTML DOM")}}
+
+The **`mediaCapabilities`** read-only property of the {{domxref("WorkerNavigator")}} interface
```
Comment (hamishwillee) [inline files/en-us/web/api/workernavigator/mediacapabilities/index.md:10]: Same as above - context of the interface is clear from the breadcrumb. 

I don't _think_ we ever distinguish these in the text, or need to.

**Review by hamishwillee — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/navigator/mediacapabilities/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/navigator/mediacapabilities/index.md:16]:
```suggestion
The **`Navigator.mediaCapabilities`** read-only property
```

File: files/en-us/web/api/workernavigator/mediacapabilities/index.md (hunk 2 above)
Comment (teoli2003) [reply files/en-us/web/api/workernavigator/mediacapabilities/index.md:10]:
```suggestion
The read-only **`WorkerNavigator.mediaCapabilities`** property
```

**Review by hamishwillee — APPROVED** [review]
Looks great.

**Pull request merged**